roles/fairfood/templates/post-receive.j2

```diff
@@ -3,7 +3,7 @@
 # bash is needed to `source` rbenv stuff
 
 # Fail on any error
-set -e
+#set -e # disabled because precompile-assets sometimes returns error exit code
```

Comment:
Why is it that precompile-assets returns error code? A good question, which I'd like to answer! I was starting by just keeping what we currently have in Prod, but will go ahead and investigate further.

I think we should `set -e` and fail on any error, because as you say that's better than trying to reload the application and loading broken code. Whatever the cause might be, it would require manual intervention.